config/scripts/check.sh

```
@@ -1105,232 +1439,1289 @@ check_disk_hotplug() {
+        if [ "${disable_sched_task_after}" = "true" ]; then
+            if [ "${ssh_ok}" != "true" ]; then
+                echo "  [13/13 optional] Disable scheduled task... SKIP (no SSH)"
+                validations+=("{\"phase\": \"disable_sched_task\", \"status\": \"SKIP\", \"message\": \"Skipped — SSH not available\"}")
+            elif [ -z "${wait_process_name}" ]; then
+                echo "  [13/13 optional] Disable scheduled task... SKIP (no waitProcessName specified)"
+                validations+=("{\"phase\": \"disable_sched_task\", \"status\": \"SKIP\", \"message\": \"waitProcessName is empty\"}")
+            else
+                echo "  [13/13 optional] Disabling scheduled task(s) matching '*${wait_process_name}*' so it will not run on the next reboot..."
+                # shellcheck disable=SC2016
+                local ps_disable_task
+                ps_disable_task='$tasks = @(Get-ScheduledTask -ErrorAction SilentlyContinue | '
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)